### ISSUE TITLE
azurerm_app_service_managed_certificate fails with Error: rpc error: code = Unavailable desc = transport is closing

### ISSUE THREAD

**Issue by protoblade** [hashicorp/terraform-provider-azurerm#13365]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v0.13.5
+ provider registry.terraform.io/alexkappa/auth0 v0.21.0
+ provider registry.terraform.io/hashicorp/azuread v2.2.1
+ provider registry.terraform.io/hashicorp/azurerm v2.76.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
### Affected Resource(s)

* `azurerm_app_service_managed_certificate`

### Terraform Configuration Files

```hcl
resource "azurerm_app_service" "webapp" {
  tags                = jsondecode(var.environment_tag)
  name                = "webapp-${var.naming}"
  location            = var.location_name
  resource_group_name = var.rsg_name
  app_service_plan_id = var.common_service_plan_id
  https_only          = true
  logs {
    detailed_error_messages_enabled = true
    failed_request_tracing_enabled  = true

    http_logs {
      file_system {
        retention_in_days = 7
        retention_in_mb   = 100
      }
    }
  }
  identity {
    type = "SystemAssigned"
  }
  site_config {
    linux_fx_version = "DOCKER|${var.common_acr_login_server}/${var.ui_acr_component_name}"
    always_on        = "true"
    ftps_state       = "Disabled"
  }
  app_settings = {
    "WEBSITES_ENABLE_APP_SERVICE_STORAGE"        = false
    "APPINSIGHTS_INSTRUMENTATIONKEY"             = var.app_insights_key
  }
  lifecycle {
    ignore_changes = [
      site_config["linux_fx_version"],
      app_settings["DOCKER_REGISTRY_SERVER_URL"],
      app_settings["WEBSITE_ENABLE_SYNC_UPDATE_SITE"],
      app_settings["WEBSITE_RUN_FROM_PACKAGE"] # prevent TF reporting configuration drift after app code is deployed
    ]
  }
}

resource "azurerm_dns_txt_record" "full_txt_record" {
  tags                = jsondecode(var.environment_tag)
  name                = "asuid.www.${var.environment}"
  zone_name           = var.common_dns_zone_name
  resource_group_name = var.common_dns_zone_rsg_name
  ttl                 = 300
  depends_on          = [azurerm_app_service.webapp]

  record {
    value = lower(azurerm_app_service.webapp.custom_domain_verification_id)
  }
}

resource "azurerm_dns_cname_record" "full_cname_record" {
  tags                = jsondecode(var.environment_tag)
  name                = "www.${var.environment}"
  zone_name           = var.common_dns_zone_name
  resource_group_name = var.common_dns_zone_rsg_name
  ttl                 = 300
  record              = azurerm_app_service.webapp.default_site_hostname
  depends_on          = [azurerm_app_service.webapp]
}

resource "azurerm_app_service_custom_hostname_binding" "full_custom_domain_binding" {
  hostname            = "www.${var.environment}.${var.common_dns_zone_name}"
  app_service_name    = "webapp-${var.naming}"
  resource_group_name = var.rsg_name
  depends_on          = [azurerm_dns_txt_record.txt_record, azurerm_dns_txt_record.full_txt_record, azurerm_dns_cname_record.full_cname_record, azurerm_dns_cname_record.cname_record]
}

resource "azurerm_app_service_managed_certificate" "full_managed_cert" {
  tags                       = jsondecode(var.environment_tag)
  custom_hostname_binding_id = azurerm_app_service_custom_hostname_binding.full_custom_domain_binding.id
}

resource "azurerm_app_service_certificate_binding" "full_cert_binding" {
  hostname_binding_id = azurerm_app_service_custom_hostname_binding.full_custom_domain_binding.id
  certificate_id      = azurerm_app_service_managed_certificate.full_managed_cert.id
  ssl_state           = "SniEnabled"
}
```

### Debug Output

https://gist.github.com/protoblade/7d4687597221ede9b5bf590abcd73c48

### Panic Output

https://gist.githubusercontent.com/protoblade/c34a7277dac08b3cb07539d2ee0dd9ed/raw/66bebdbb60f6cdf85de513ee4bdd97f55651c5ce/gistfile1.txt

### Expected Behaviour

Terraform apply to work (it actually did without any changes until 2 days ago)

### Actual Behaviour

Terraform exits with `Error: rpc error: code = Unavailable desc = transport is closing`
on second apply exits with already exists - to be managed via Terraform this resource needs to be imported into the State.
on import crashes (crash log added)

Terraform crashes with panic

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

no
### References

* #0000

**Comment by aristosvo** [comment]
Hi @protoblade! This is a duplicate of issue #13353, which is fixed and planned to be released tomorrow

**Comment by jackofallops** [comment]
Closed by #13357
